README.md

```diff
@@ -131,6 +131,8 @@ _All product names, logos, and brandsare property of their respective owners. Al
 | <img height="50" src="https://user-images.githubusercontent.com/25181517/187896150-cc1dcb12-d490-445c-8e4d-1275cd2388d6.png"> |      Redux      | `https://user-images.githubusercontent.com/25181517/187896150-cc1dcb12-d490-445c-8e4d-1275cd2388d6.png` |
 | <img height="50" src="https://user-images.githubusercontent.com/25181517/187955005-f4ca6f1a-e727-497b-b81b-93fb9726268e.png"> |      Jest       | `https://user-images.githubusercontent.com/25181517/187955005-f4ca6f1a-e727-497b-b81b-93fb9726268e.png` |
 | <img height="50" src="https://user-images.githubusercontent.com/25181517/187955008-981340e6-b4cc-441b-80cf-7a5e94d29e7e.png"> |     webpack     | `https://user-images.githubusercontent.com/25181517/187955008-981340e6-b4cc-441b-80cf-7a5e94d29e7e.png` |
+| <img height="50" src="https://res.cloudinary.com/codelikeagirl29/image/upload/v1670293836/icons/gatsby-plain_ragmxp.svg"> |      Gatsby       | `https://res.cloudinary.com/codelikeagirl29/image/upload/v1670293836/icons/gatsby-plain_ragmxp.svg` |
```

Comment:
Second thing is that we can try to have all icons as png and 400x400 pixels, I have notice that if we would keep large png the load time of profile increase what makes it less atractive.

What do you think about it?

I do not want to west your time, so I can scale this img by myself :)